### PR TITLE
added voting functionality to each article

### PR DESCRIPTION
**Pull request by DavidH3031**

### CODE REVIEW

File: src/components/SingleArticle.jsx [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
+        } else {
+          setErr(null);
+          setClicked(true);
+        }
+      });
+      setVotes((currVotes) => {
+        return currVotes + 1;
+      });
+    } else {
+      updateVotes(article_id, 1).then((res) => {
```
Comment (callumu21) [inline src/components/SingleArticle.jsx]: Looks like you're sending an upvote here but displaying a downvote to the user.

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -33,6 +72,22 @@ function SingleArticle() {
...
+              </button>
+              <button
+                className="voting-buttons"
+                onClick={!clicked ? handleClick : null}
+              >
+                👎
+              </button>
+            </section>
+            <h3 className="upvotes">Upvotes: {votes}</h3>
+            <h3 className="error-msg">{err}</h3>
```
Comment (callumu21) [inline src/components/SingleArticle.jsx:93]: Very nice - good consideration of user experience and feedback

**Review by callumu21 — CHANGES_REQUESTED** [review]
Good start, and great consideration of error messages. However, your downvote functionality doesn't work at the moment as it actually increments the vote.

It could also be good to provide the user with some visual feedback on the button being pressed - maybe try disabling them once a vote has been made, or highlighting the one that has been pressed?

**Comment by JVal-NC** [comment]
Looks good go ahead and merge.

**Pull request merged**